org.eclipse.m2e.core.tests/build.properties

```
@@ -3,3 +3,7 @@ output.. = bin/
 bin.includes = META-INF/,\
                resources/,\
                .
+# Maven Properties, see: https://github.com/eclipse/tycho/wiki/Tycho-Pomless#define-properties
+pom.model.property.testClass = org.eclipse.m2e.core.AllTests
+pom.model.property.tycho.surefire.useUIHarness = false
```

it *might be required to use
```suggestion
pom.model.property.useUIHarness = false
```

(same below)